docs/SANITY.md

```diff
@@ -1,13 +1,35 @@
-## Tipado automático de consultas de Sanity
+<div align="center" width="100%">
+    <h1>La Cuentoneta</h1>
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://github.com/rolivencia/cuentoneta/assets/32349705/b0ea0659-3c9d-4c4f-9d14-ab60d50dd832">
+        <img width="33%" alt="La Cuentoneta" src="https://github.com/rolivencia/cuentoneta/assets/32349705/b0ea0659-3c9d-4c4f-9d14-ab60d50dd832">
+    </picture>
+</div>
 
-Para generar los tipos de las consultas de Sanity debemos seguir [esta guía](https://www.sanity.io/learn/course/typescripted-content/generating-type-for-groq-query-results). El link lleva directamente al paso 5 que explica cómo generar los tipos para una query GROQ. Los pasos anteriores son de configuración y no son necesarios para el ya que ya fueron completados (el `schema.json` puede encontrarse en `cms/schema.json`, por ejemplo). Solo en caso de que agreguemos nuevas entidades necesitaremos actualizarlo.
+# Tipado automático de consultas de Sanity (GROQ)
 
-De acuerdo a lo definido en `cms/sanity-typegen.json` los tipos generados mediante `sanity typegen generate` pueden encontrarse en `src/api/sanity/types.ts`, hacia el final del archivo.
+Para generar los tipos de las consultas de Sanity debemos seguir [esta guía](https://www.sanity.io/learn/course/typescripted-content/generating-type-for-groq-query-results). El link lleva directamente al paso 5 que explica cómo generar los tipos para una query GROQ. Los pasos anteriores son de configuración y no son necesarios para el ya que ya fueron completados (el `schema.json` puede encontrarse en `cms/schema.json`, por ejemplo). Solo en caso de que agreguemos nuevas entidades o se agreguen modificaciones en los schemas de entidades existentes necesitaremos actualizarlo.
```

Comment:
Creo que se puede resumir un poco, en lugar de:

Para generar los tipos de las consultas de Sanity debemos seguir [esta guía](https://www.sanity.io/learn/course/typescripted-content/generating-type-for-groq-query-results). El link lleva directamente al paso 5 que explica cómo generar los tipos para una query GROQ. Los pasos anteriores son de configuración y no son necesarios para el ya que ya fueron completados (el `schema.json` puede encontrarse en `cms/schema.json`, por ejemplo). Solo en caso de que agreguemos nuevas entidades o se agreguen modificaciones en los schemas de entidades existentes necesitaremos actualizarlo.

Usar:

Para aprender a generar los tipos para una query GROQ de Sanity debemos seguir [esta guía](https://www.sanity.io/learn/course/typescripted-content/generating-type-for-groq-query-results). Solo en caso de que agreguemos nuevas entidades o se agreguen modificaciones en los schemas de entidades existentes necesitaremos actualizar (el `schema.json` puede encontrarse en `cms/schema.json`, por ejemplo).

Reply:
Actualizado este párrafo. Gracias @ErikGIovani.